src/Definition/Argument.php

```diff
@@ -87,4 +87,12 @@ public function count(): int
     {
         return count($this->rawArguments);
     }
+
+    /**
+     * @return mixed
+     */
+    public function __get(string $name)
```

Comment:
Argument is an implementation of ArgumentInterface. This PR break the possibility to use custom Argument implementing ArgumentInterface.

Reply:
I can add `__get` method to the interface too.